### ISSUE TITLE
MalisisDoors and OptiFine incompatabilities

### ISSUE THREAD

**Issue by ghost** [Ordinastie/MalisisDoors#264]
Minecraft version: 1.12.2
Forge version: 14.23.2707
MalisisDoors version: 7.3.0

When I run MalisisDoors alongside OptiFine, certain things don't render.

The frame of the rusty hatch:

![image](https://user-images.githubusercontent.com/20571660/40869840-b200622c-65e7-11e8-85b9-605cea2eeb40.png)

The door factory:

![image](https://user-images.githubusercontent.com/20571660/40869844-c95d1456-65e7-11e8-9b06-6a2d20f40231.png)

The rusty ladder: 

Notice the outline around where the ladder is supposed to be rendered.

![image](https://user-images.githubusercontent.com/20571660/40869854-10723c90-65e8-11e8-93e2-c5407409d99e.png)

**Comment by Xtreemo** [comment]
That's true. They can't render with me as well. Good thing this issue is not only happening to me.

**Comment by Stuethekid** [comment]
i was about to leave a bug report about this

**Comment by tusharaggarwa1** [comment]
The same issue also happens for the majority of items from MalisisBlocks as well.

**Comment by Draskh** [comment]
I had already reported that. Seems that there is no pogress. This could be closed then also,
https://github.com/Ordinastie/MalisisDoors/issues/209

**Comment by Draskh** [comment]
Note: This is happening with Optifine E2 and Forge 2765 also.

**Comment by OmegaPaladin** [comment]
This is still occurring with Optifine U3 and Forge 2795.  Has anyone tried modifying settings for Optifine?

This occurs with Malisis switches as well.

**Comment by JavaKitsune** [comment]
I know this is an old thread but I was able to fix this problem occurring with the Door Factory being invisible and the Rusty Ladder. It is due to Optifine's rendering and I turned off Render Regions in Options > Video Settings > Performance. 

Door Factory and Rusty Ladder were not invisible anymore but did not fix the Rusty Hatch.

**Comment by MrRedstoneA1** [comment]
> I know this is an old thread but I was able to fix this problem occurring with the Door Factory being invisible and the Rusty Ladder. It is due to Optifine's rendering and I turned off Render Regions in Options > Video Settings > Performance.
> 
> Door Factory and Rusty Ladder were not invisible anymore but did not fix the Rusty Hatch.

Thank you, I've been looking for a fix for close to an hour with various search terms. This solution needs to be shared across everything. It didn't fix the hatch for me as well, but everything else renders properly now.

**Comment by SrToteco** [comment]
@JavaKitsune What version of forge and optifine are you using?
